iohub/ngff/nodes.py

```diff
@@ -1010,6 +1010,43 @@ def get_axis_index(self, axis_name: str) -> int:
         """
         return self.axis_names.index(axis_name.lower())
 
+    def get_transforms(
+        self,
+        image: str | Literal["*"],
+    ) -> tuple[TransformationMeta, TransformationMeta]:
+        """Get the total coordinate scale and translation metadata
+        for one image array or the whole FOV.
+
+        Parameters
+        ----------
+        image : str | Literal["*"]
+            Name of one image array (e.g. "0") to transform,
+            or "*" for the whole FOV
+        """
+        transforms: list[TransformationMeta] = self.metadata.multiscales[0].coordinate_transformations
+        if image != "*" and image in self:
+            for i, dataset_meta in enumerate(self.metadata.multiscales[0].datasets):
+                if dataset_meta.path == image:
+                    transforms.extend(
+                        self.metadata.multiscales[0]
+                        .datasets[i]
+                        .coordinate_transformations
+                    )
+        elif image != "*":
+            raise ValueError(f"Key {image} not recognized.")
+
+        full_scale = np.array([1]*len(self.axes), dtype=float)
+        full_translation = np.array([0]*len(self.axes), dtype=float)
+        for transform in transforms:
+            if transform.type == "scale":
+                full_scale *= np.array(transform.scale)
+            elif transform.type == "translation":
+                full_translation += full_scale * np.array(transform.translation)
```

Comment:
Does translation need to be scaled here? This would be necessary if the translation is given in pixel space, and needs to be converted into physical units. However the [spec](https://ngff.openmicroscopy.org/0.4/index.html#multiscale-md) says that they should already be in physical units (e.g. microns):

```text
It MAY contain exactly one translation that specifies the offset from the origin in physical units. If translation is given it MUST be listed after scale to ensure that it is given in physical coordinates.
```

Reply:
Ah ok. So I got it backwards. It is still a bit unclear to me what happens if the multiscale group has `[shift(5), translate(5)]` and the individual image "0" has `[shift(2), translate(2)]`. Is the effective scale 10, and the effective translation 7? That seems to make sense if its just assumed that we are never dealing with voxels, we are always in physical unit space when defining translations

Reply:
Yeah sorry, I meant scale not shift. I fixed it